content/en/cloud_cost_management/google_cloud.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Google Cloud
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/aws"
+  tag: "Documentation"
+  text: "Gain insights into your AWS bill"
+- link: "/cloud_cost_management/azure"
+  tag: "Documentation"
+  text: "Gain insights into your Azure bill"
+---
+
+{{< beta-callout url="https://docs.google.com/forms/d/e/1FAIpQLScNSzkTAmmpmxEY5Xoo-xPOyKXG0n_6IgdlU66mWXp445pNdA/viewform" btn_hidden="false" >}}
+Cloud Cost Google Cloud support is in private beta
+{{< /beta-callout >}}
+
+## Overview
+
+To use Cloud Cost Management with Google Cloud, you must set up a Datadog Google Cloud Platform Integration in Datadog. You must also enable a Detailed Usage Cost billing export for your desired billing account in Google Cloud.
+
+The Google Cloud Platform Datadog Integration Service Account must have necessary permissions configured to interact with Google Cloud Storage and BigQuery.
+
+## Setup
+
+### Enable Detailed Usage cost billing export
+<div class="alert alert-info"> <a href="https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage">Detailed Usage Cost</a> data includes resource-level cost data. </div>
+
+1. Navigate to [Billing Export][1] under Google Cloud console *Billing*
+2. Enable [*Detailed Usage cost*][2] export, selecting or creating a project and a BigQuery dataset
+3. Document the billing account id for the billing account where the export was configured as well as export project id and dataset name
+
+### Create or select a Google Cloud Storage bucket
+You can use an existing GCS bucket or create a new one. Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to a `datadog_cloud_cost_detailed_usage_export` prefix in the selected bucket.
+
+**Note:** The bucket [must be co-located][9] with the BigQuery export dataset.
```

Comment:
```suggestion
**Note:** The bucket [must be co-located][9] within the BigQuery export dataset.
```

Reply:
I think this should stay as `with` since the bucket and bigquery export are independent - the bucket won't be in the BigQuery dataset but has to be in the same region with the bigquery dataset 

I wasn't sure if I should use co-locate/collocate/colocate as the right term - recommendations seemed to vary online